### ISSUE TITLE
Add graduation reports for Activity Stream, Page Shot, Pulse, and Tab Center

### ISSUE THREAD

**Issue by pdehaan** [mozilla/testpilot#2690]
Filing this for my own tracking. Plus, it may be nice to get this assigned into a milestone.

Currently the following four recently graduated experiments all say:

> Experiment End Date: ⁨7/12/2017⁩
> ...
> **This experiment has ended**
> We are working on a full report. Check back soon for the details.

- https://testpilot.firefox.com/experiments/activity-stream
- https://testpilot.firefox.com/experiments/page-shot
- https://testpilot.firefox.com/experiments/pulse
- https://testpilot.firefox.com/experiments/tab-center

Once these land in dev (or a PR), I can verify the images and HTML markup and all that stuff.

**Comment by johngruen** [comment]
Closing this in favor of #2694
